lisa/schema.py

```diff
@@ -774,9 +774,9 @@ def check(self, capability: Any) -> search_space.ResultReason:
             search_space.check_countspace(self.memory_mb, capability.memory_mb),
             "memory_mb",
         )
-        if self.disk:
+        if self.disk and capability.disk:
```

Comment:
Can you share the detail error? I'm not sure if it hides a bug underlying.

Reply:
Thank you for the shared log. As the design, the capability must be filled by platform, it shouldn't be skipped, if it's not presented. As the code here, `self.disk` is the requirement to the disk, if it's None, it means there is no requirement on disk. But the `capability.disk` must be presented for comparison. The improvement here is,

1. Add another assertion here to let developer know, the disk/network capability of an environment must be presented.
2. Add capability in the prepare_environment in the platform. You can start with a very basic capability, so it won't fail the test cases, which has disk requirements.